### DISCUSSION TITLE
右侧日历图小组件设置

### DISCUSSION THREAD

**Discussion by webyang-male** [Tokinx/Wing#106] [问答]
如题，请问博主是怎么设置主题展示站点右侧的日历图小组件，麻烦您了。

**Comment by ZhangWeilong** [comment]
![image](https://github.com/Tokinx/Wing/assets/8246653/5e9af23e-11fe-439f-a992-4d83750e1cd9)

你说的是这个东西吗？这个是创建的笔记页面，之后设置成首页就可以了。

**Reply by webyang-male** [reply]
是的，谢谢您
